samples/java/agents/magic_8_ball_security/README.md

```diff
@@ -0,0 +1,117 @@
+# Magic 8-Ball Security Agent
+
+This sample agent responds to yes/no questions by consulting a Magic 8-Ball.
+
+This agent demonstrates how to secure an A2A server with Keycloak using bearer token authentication and it shows how to configure an A2A client to specify the token when
+sending requests. The agent is written using Quarkus LangChain4j and makes use of the
+[A2A Java](https://github.com/a2aproject/a2a-java) SDK.
+
+## Prerequisites
+
+- Java 17 or higher
+- Access to an LLM and API Key
+
+## Running the Sample
+
+This sample consists of an A2A server agent, which is in the `server` directory, and an A2A client,
+which is in the `client` directory.
+
+### Running the A2A Server Agent
+
+1. Navigate to the `magic-8-ball-security` sample directory:
+
+    ```bash
+    cd samples/java/agents/magic-8-ball-security/server
+    ```
+
+2. Set your Google AI Studio API Key as an environment variable:
+
+   ```bash
+   export QUARKUS_LANGCHAIN4J_AI_GEMINI_API_KEY=your_api_key_here
+   ```
+
+   Alternatively, you can create a `.env` file in the `magic-8-ball-security/server` directory:
+
+   ```bash
+   QUARKUS_LANGCHAIN4J_AI_GEMINI_API_KEY=your_api_key_here
+   ```
+
+3. Start the A2A server agent
+
+   **NOTE:**
+   By default, the agent will start on port 11000. To override this, add the `-Dquarkus.http.port=YOUR_PORT`
+   option at the end of the command below.
+
+   ```bash
+   mvn quarkus:dev
+   ```
+
+### Running the A2A Java Client
+
+The Java `TestClient` communicates with the Magic 8-Ball Agent using the A2A Java SDK.
+
+The client supports specifying which transport protocol to use ("jsonrpc", "rest", or "grpc"). By default, it uses JSON-RPC.
+
+1. Make sure you have [JBang installed](https://www.jbang.dev/documentation/guide/latest/installation.html)
```

Comment:
This page does not seem to exist :-) I found https://www.jbang.dev/download/

Reply:
https://www.jbang.dev/documentation/jbang/latest/installation.html is working for me ;)

Reply:
guide should be replaced with jbang

Reply:
Fixed